utility/install_node.sh

```diff
@@ -14,18 +14,7 @@ case "${dpkgArch##*-}" in \
   esac
   # gpg keys listed at https://github.com/nodejs/node#release-keys
   set -ex
-  for key in \
-      94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
-      FD3A5288F042B6850C66B31F09FE44734EB7990E \
-      71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
-      DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-      C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
-      B9AE9905FFD7803F25714661B63B535A4C206CA9 \
-      77984A986EBC2AA786BC0F66B01FBB92821C587A \
-      8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
-      4ED778F539E3634C779C87C6D7062848A1AB005C \
-      A48C2BEE680E841632CD4E44F07496B3EB3C1762 \
-      B9E2F5981AA6E0CD28160D9FF13993A75599653C \
+  for key in $(curl -sL https://raw.githubusercontent.com/nodejs/docker-node/master/keys/node.keys) \
```

Comment:
I don't fully understand the implications here. Without this, I was getting an error building the Docker image:

```
gpg: Can't check signature: No public key
```

I found this solution at: https://github.com/nodejs/unofficial-builds/pull/39/files

Reply:
@jgerigmeyer Can we update the list to the current contents of that file rather than making the script download the file? That way we aren't vulnerable if someone manages to take over that repository and replace the keys.